share/doc/xml/chandra_repro.xml

```diff
@@ -733,6 +733,20 @@ acisf084244478N003_2_bias0.fits.gz
       
       </PARAM>
 
+      <PARAM name="pi_filter" def="yes" type="boolean" reqd="no">
+         <SYNOPSIS>
+          Should the additional PI filter be applied to HRC-S+LETG data?
+         </SYNOPSIS>
+         <DESC>
+            <PARA>
+                This parameter controls whether the optional HRC-S+LETG
+                PI filter should be used.  The PI filter is designed 
+                to suppress the background with only minor effect on
+                the source spectrum.
+            </PARA>
```

Comment:
Can we add a paragraph to explain when you might want to turn this off - e.g. such as the current situation with gain shenanighans (which I don't know enough to explain)?